test/fixtures/unicorn/prefer-add-event-listener.pass.ts

```diff
@@ -0,0 +1,3 @@
+/**
+ * TODO: Find a good way to test this
```

Comment:
Should this be expiring? Or maybe we can just test it?

Reply:
Fixed with: https://github.com/ridedott/eslint-config/pull/319/commits/09dc2a5dc7793d9447eb5b3402b77e064cece216